http/base/src/main/java/org/wildfly/security/http/HttpConstants.java

```diff
@@ -182,6 +182,7 @@ private HttpConstants() {
     public static final int BAD_REQUEST = 400;
     public static final int UNAUTHORIZED = 401;
     public static final int FORBIDDEN = 403;
+    public static final int NOT_FOUND = 404;
```

Comment:
Sorry, there was a typo in the issue description so this 404 constant doesn't need to be introduced here. We'll need to check for UNAUTHORIZED, as shown below.